interfaces/cython/cantera/ck2cti.py

```diff
@@ -1735,10 +1727,10 @@ def readline():
                                 try:
                                     label, thermo, comp, note = self.readThermoEntry(thermo, TintDefault)
                                 except Exception as e:
-                                    print('Error while reading thermo entry '
-                                        'starting on line {0}:\n"""\n{1}\n"""'.format(
-                                            self.line_number-len(current)+1,
-                                            ''.join(current).rstrip()))
+                                    logging.error('Error while reading thermo entry '
+                                                  'starting on line {0}:\n"""\n{1}\n"""'.format(
+                                                   self.line_number-len(current)+1,
```

Comment:
I can't say I'm a fan of this indentation (specifically, the additional 1 space inside the `format(...)`, although admittedly most of the alternatives aren't great either.